src/pages/library/DataBrowserPreview.js

```diff
@@ -136,7 +136,7 @@ const DataBrowserPreview = ({ id }) => {
         div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'top', justifyContent: 'space-between', width: '100%', lineHeight: '26px' } }, [
           h1([snapshot['dct:title']]),
           h(Link, {
-            href: Nav.getLink('library-details', { id: Nav.getCurrentRoute().params.id }),
+            onClick: Nav.history.goBack,
             'aria-label': 'Close',
```

Comment:
Why is the aria label for this Close?

Reply:
I think @lu-c or @petesantos would have more context about that

Reply:
I'm trying to understand if it should be "Back" instead of "Close" since the action is to go back in the history. But I'm also having trouble understanding in the video which button this is.

Reply:
This doesn't look right - we should change this to 'go back'.

Reply:
Having watched this video a few more times, I think this is dismissing the Data Preview view and going back to the Data Set details. So I can see "Close" given that the icon is an X, and perhaps the fact that it is implemented as going back in the history isn't that important. What do you think @petesantos?

(It seems like this might be better as a Back (arrow) icon instead of X, but I am still new to how we manage/expose our internal navigation)

Reply:
A back arrow icon with the aria-label "go back" would make it consistent with the details page, so that makes sense to me. What do you think @petesantos?

Reply:
After discussing this with my team, the consensus was that we can leave it as an "X" for now since users haven't been confused by this behavior, but we can revisit later if needed.